shared/cfg/guest-hw.cfg

```diff
@@ -114,6 +114,9 @@ variants:
         # placeholder
 
 variants:
+    - luks:
+        image_format = luks
```

Comment:
Hi guy
it looks that LUKS is a disk encryption specification, not a image format, right?

Reply:
No, there is .luks image format.
Please refer to
https://qemu.weilnetz.de/doc/qemu-doc.html#disk_005fimages_005fformats

eg:
>     #qemu-img create -f luks --object secret,id=sec0,data=`printf %s
> "redhat" |base64`,format=base64 -o key-secret=sec0 demo.luks 5G
>     #qemu-img info demo.luks 
>     image: demo.luks
>     file format: luks
>     virtual size: 5.0G (5368709120 bytes)
>     disk size: 256K
>     encrypted: yes